src/pyFAI/gui/utils/tree.py

```diff
@@ -35,9 +35,8 @@
 __status__ = "development"
 __docformat__ = 'restructuredtext'
 
-import os
 import logging
-from ...io.diffmap_config import DataSet, DataSetNT, ListDataSet
-from ...io.tree import TreeItem
+from ...io.diffmap_config import DataSet as DataSet, DataSetNT as DataSetNT, ListDataSet as ListDataSet
+from ...io.tree import TreeItem as TreeItem
```

Comment:
Some unwanted crazy automated changes I guess 😁 

```suggestion
from ...io.diffmap_config import DataSet, DataSetNT, ListDataSet
from ...io.tree import TreeItem
```

Reply:
Side effect of using "modern" code checkers / formatters (here `ruff`) which enforce explicit re-export.
Did I tell you I was not a fan of those tools :-)